app/src/js/components/Rules/rule.js

```diff
@@ -27,15 +27,31 @@ import Loading from '../LoadingIndicator/loading-indicator';
 import Metadata from '../Table/Metadata';
 import AsyncCommands from '../DropDown/dropdown-async-command';
 import ErrorReport from '../Errors/report';
+import Breadcrumbs from '../Breadcrumbs/Breadcrumbs';
+
+const breadcrumbConfig = [
+  {
+    label: 'Dashboard Home',
+    href: '/'
+  },
+  {
+    label: 'Rules',
+    href: '/Rules'
+  },
+  {
+    label: 'Rule Overview',
+    active: true
+  }
+];
 
 const metaAccessors = [
-  ['RuleName', 'name'],
+  ['Rule Name', 'name'],
+  ['Timestamp', 'timestamp', fullDate],
```

Comment:
Wondering if instead of timestamp we want to be looking at `createdAt` and/or `updatedAt` since it seems like timestamp is empty.

Reply:
The `metaAccessors` code was originally there. I was just making sure things were spelled right or the content made sense. I think `updatedAt` would be good since it is existing to add to the implementation ticket.

Reply:
Fair enough, makes sense!